contributing-guides/style-guide.md

```diff
@@ -104,6 +104,12 @@ Example:
 ```
 - Pre-translated alias page templates can be found [here](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/alias-pages.md).
 
+## Description syntax
+
+- Use double quotes for quoting.
+- Quote just strings.
```

Comment:
Sorry, I can't say I understand what this means. :thinking: 
Could you elaborate in the comments?

Reply:
It means that when you wanna quote smth in description always use double quotes. For instance what if I wanna mention some string literal used in code example? I should write `"some literal text here"` and not with single quotes.

Reply:
Ahh, sorry! I meant more specifically: "Quote just strings."
I understood the goal of this was to specify the use of double-quotes, but don't understand what is meant by "Quote just strings."

Reply:
Is wording better now?